Imaging_scripts_HB.txt

```diff
@@ -0,0 +1,44 @@
+Here is a description of the imaging-related MATLAB scripts.
```

Comment:
This seems like the instructions for this repo. This could be made into a README.md document.